gen/libdir/forbidden.conf

```
@@ -100,51 +108,80 @@
 # let ctrl = [Exp & R]; basic-dep; [BCC]; po
 # let IFB-ob = [Exp & R]; ctrl; [IFB]; po
 # let IFB = ISB | EXC-ENTRY-IFB | EXC-RET-IFB
--safe [DpCtrldR ISB] [DpCtrldW ISB] [DpCtrlsR ISB] [DpCtrlsW ISB]
+-safe [DpCtrldR ISB] [DpCtrldW ISB] [DpCtrlsR ISB]
+# [DpCtrlsW ISB] is stronger than `Pos*W`
+#-safe [DpCtrlsW ISB]
 
 # let IFB-ob = [Exp & R]; pick-ctrl-dep; [IFB]; po
 # let IFB = ISB | EXC-ENTRY-IFB | EXC-RET-IFB
--safe [DpCtrlCseldR ISB] [DpCtrlCseldW ISB] [DpCtrlCselsR ISB] [DpCtrlCselsW ISB]
+-safe [DpCtrlCseldR ISB] [DpCtrlCseldW ISB] [DpCtrlCselsR ISB]
+# [DpCtrlCselsW ISB] is stronger than `Pos*W`
+#-safe [DpCtrlCselsW ISB]
 
 # let IFB-ob = [Exp & R]; addr; [Exp & M]; po; [IFB]; po
--safe [DpAddrdR ISBd**] [DpAddrdR ISBs**] [DpAddrdW ISBd**] [DpAddrdW ISBs**] [DpAddrsR ISBd**] [DpAddrsR ISBs**] [DpAddrsW ISBd**] [DpAddrsW ISBs**]
+-safe [DpAddrdR ISBdR*] [DpAddrdR ISBsR*] [DpAddrdW ISBdW*] [DpAddrdW ISBsW*] [DpAddrsR ISBdR*] [DpAddrsR ISBsRR] [DpAddrsW ISBdW*] [DpAddrsW ISBsWR]
+# `[DpAddrsW ISBsWW] [DpAddrsR ISBsRW]` are stronger than `Pos*W` as the former are a read followed by a write to the same location.
+#-safe [DpAddrsW ISBsWW] [DpAddrsR ISBsRW]
+
 # let IFB-ob = [Exp & R]; pick-addr-dep; [Exp & M]; po; [IFB]; po
--safe [DpAddrCseldR ISBd**] [DpAddrCseldR ISBs**] [DpAddrCseldW ISBd**] [DpAddrCseldW ISBs**] [DpAddrCselsR ISBd**] [DpAddrCselsR ISBs**] [DpAddrCselsW ISBd**] [DpAddrCselsW ISBs**]
-#
+-safe [DpAddrCseldR ISBdR*] [DpAddrCseldR ISBsR*] [DpAddrCseldW ISBdW*] [DpAddrCseldW ISBsW*] [DpAddrCselsR ISBdR*] [DpAddrCselsR ISBsRR] [DpAddrCselsW ISBdW*] [DpAddrCselsW ISBsWR]
+# `[DpAddrCselsW ISBsWW] [DpAddrCselsR ISBsRW]` are stronger than `Pos*W` as the former are a read followed by a write to the same location.
+#-safe [DpAddrCselsW ISBsWW] [DpAddrCselsR ISBsRW]
+
 # let pob = pick-addr-dep; [Exp & W | HU | TLBI | DC.CVAU | IC]
 # mcat2config7 aarch64.cat -let pob
--safe DpAddrCseldW DpAddrCselsW
+-safe DpAddrCseldW
+# `DpAddrCselsW` is stronger than `Pos*W`.
+#-safe DpAddrCselsW
 
 # let pob = pick-data-dep
--safe DpDataCseldW DpDataCselsW
+-safe DpDataCseldW
+# `DpDataCselsW` is stronger than `Pos*W`.
+#-safe DpDataCselsW
 
 # let pob = pick-ctrl-dep; [Exp & W | HU | TLBI | DC.CVAU | IC]
--safe DpCtrlCseldW DpCtrlCselsW
+-safe DpCtrlCseldW
+# `DpCtrlCselsW` is stronger than `Pos*W`.
+#-safe DpCtrlCselsW
 
 # let pob = pick-addr-dep; [Exp & M]; po; [Exp & W | HU]
--safe [DpAddrCseldR PodRW] [DpAddrCseldR PosRW] [DpAddrCseldW PodWW] [DpAddrCseldW PosWW] [DpAddrCselsR PodRW] [DpAddrCselsR PosRW] [DpAddrCselsW PodWW] [DpAddrCselsW PosWW]
+-safe [DpAddrCseldR PodRW] [DpAddrCseldR PosRW] [DpAddrCseldW PodWW] [DpAddrCseldW PosWW] [DpAddrCselsR PodRW] [DpAddrCselsW PodWW]
```

Because DpAddrCseldW and Pos*W are already included, I think [DpAddrCseldW PosWW] is unnecessary
```suggestion
-safe [DpAddrCseldR PodRW] [DpAddrCseldR PosRW] [DpAddrCseldW PodWW] [DpAddrCselsR PodRW] [DpAddrCselsW PodWW]
```